Wrapper/Wrapper.php

```diff
@@ -88,7 +89,7 @@ public static function register($protocol, $className, $flags = 0)
      */
     public static function unregister($protocol)
     {
-        return stream_wrapper_unregister($protocol);
+        return @stream_wrapper_unregister($protocol);
```

Comment:
This function already returns `false` if `$protocol` does not exist, but an error is triggered. This is not the expected behavior for our API. I will add an inline explanation.